catppuccin-theme.el

```diff
@@ -939,7 +939,6 @@ FLAVOR defaults to the value of `catppuccin-flavor'."
          (magit-process-ok :foreground ,ctp-green :weight bold)
 
          ;; markdown
-         (markdown-blockquote-face :foreground ,ctp-green)
          (markdown-blockquote-face :extend t :background ,ctp-mantle
            :foreground ,ctp-green
            ,@(when catppuccin-italic-blockquotes '(:slant italic)))
```

Comment:
Could you change this definition of markdown-blockquote-face to just `:inherit org-quote` to avoid duplicating the whole list?

Reply:
This creates the same problem in reverse—if using Markdown before Org is loaded, then Markdown quotes won't be styled correctly. Perhaps we could define a new face and have both `markdown-blockquote-face` and `org-quote` inherit from it?

Reply:
Ah right. In that case I think your current solution is fine, an extra face is too much hassle. :+1: